src/StatsModels.jl

```diff
@@ -2,33 +2,60 @@ __precompile__(true)
 
 module StatsModels
 
-using Compat
-using DataFrames
+using Tables
 using StatsBase
-using Compat.SparseArrays
-using Compat.LinearAlgebra
-using Compat: @debug, @warn
+using CategoricalArrays
+using DataStructures
 
-export @formula,
-       Formula,
-       ModelFrame,
-       ModelMatrix,
+using SparseArrays
+using LinearAlgebra
+using Missings
 
-       AbstractContrasts,
-       EffectsCoding,
-       DummyCoding,
-       HelmertCoding,
-       ContrastsCoding,
+using Tables: ColumnTable
+using StatsBase: StatisticalModel
 
-       coefnames,
-       dropterm,
-       setcontrasts!
+export
+    @formula,
+    ModelFrame,
+    ModelMatrix,
 
+    AbstractContrasts,
+    EffectsCoding,
+    DummyCoding,
+    HelmertCoding,
+    ContrastsCoding,
+
+    coefnames,
+    dropterm,
+    setcontrasts!,
+
+    AbstractTerm,
+    TermOrTerms,
+    ConstantTerm,
+    Term,
+    ContinuousTerm,
+    CategoricalTerm,
+    InteractionTerm,
+    FormulaTerm,
+    InterceptTerm,
+    FunctionTerm,
+
```

Comment:
Did you intend to export `MatrixTerm`?
```suggestion
    MatrixTerm,
```

Reply:
in fact I did export it and then forgot to push :)